components/infieldbutton/metadata/infieldbutton.yml

```diff
@@ -22,7 +26,7 @@ examples:
           <h4 class="spectrum-Heading spectrum-Heading--sizeXS spectrum-Examples-itemHeading">S</h4>
           <button class="spectrum-InfieldButton spectrum-InfieldButton--sizeS spectrum-InfieldButton--right" aria-haspopup="listbox">
```

Comment:
A little hard to tell this abstracted from it's final use but is `aria-haspopup="listbox"` the correct aria to use in this case? 

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-haspopup

Reply:
So this was already there but I left it in because these buttons are typically used to display a select menu with a list of options of some sort, which would warrant the use of `aria-haspopup`. It looks like `listbox` should be used for "lists from which a user may select one or more items" so I think it makes sense here but that obviously depends on how the user implements it